tests/test_visitors/test_ast/test_naming/test_variable_self_reassignment.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+from wemake_python_styleguide.violations.best_practices import (
+    ReassigningVariableToItselfViolation,
+)
+from wemake_python_styleguide.visitors.ast.naming import (
+    WrongVariableAssignmentVisitor,
+)
+
+wrong_fragment = """
```

Comment:
Can you please also test this example:

```python
test_variable = 5
test_variable = test_variable = 5
```

and

```python
test_variable = 5
test_variable = other = test_variable = 5
```

All these examples should raise a violation.

Reply:
Fixed